.scripts/update_year.py

```
@@ -0,0 +1,127 @@
+def update_license_file(license_path, old_year, new_year):
+    """Update copyright year in LICENSE files."""
+    if not os.path.exists(license_path):
+        return False
+
+    with open(license_path, 'r', encoding='utf-8') as f:
+        content = f.read()
+
+    original_content = content
+
+    # Update copyright years: 2019–2025 -> 2019–2026, or any year range ending with old_year
+    content = re.sub(rf'(\d{{4}})–{old_year}', rf'\1–{new_year}', content)
```

Ruff (0.14.10)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)